server/controllers/Contacts.js

```diff
@@ -0,0 +1,67 @@
+const db = require("../models");
+const Contact = db.Contact;
+const SigninOption = db.SigninOption;
+const User = db.User;
+
+// Retrieve all Contact relationships in the database
+module.exports.getAllContacts = async (req, res) => {
+    let contacts = await Contact.findAll();
+    return res.send(contacts);
+}
+
+// Create and Save a new Contact
+module.exports.addContact = async (req, res) => {
+    let user_id = req.params.user_id;//temporarily getting the user_id from the url
```

Comment:
nit: 
```suggestion
    // temporarily getting the user_id from the url
    let user_id = req.params.user_id;
```

Reply:
Why is it temporary?

Reply:
I thought we'd pass it in a more secure way in the future

Reply:
After thinking about it, this wouldn't be temporary, but instead we'd have to do a check to make sure the requesting user's id matches the user_id in the url